spec/view_helper/description_spec.rb

```diff
@@ -47,6 +47,12 @@
     end
   end
 
+  it 'should change double quote to single quote' do
+    subject.display_meta_tags(:site => 'someSite', :description => 'some "description"').tap do |meta|
```

Comment:
Line is too long. [102/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.